aesara/tensor/elemwise_cgen.py

```diff
@@ -125,20 +145,41 @@ def make_alloc(loop_orders, dtype, sub, fortran="0"):
     if type.startswith("AESARA_COMPLEX"):
         type = type.replace("AESARA_COMPLEX", "NPY_COMPLEX")
     nd = len(loop_orders[0])
-    init_dims = ""
     # For each dimension, the tensors are either all broadcasted, in
     # which case the output will also be broadcastable (dimension =
     # 1), or one or more are not broadcasted, in which case the number
     # of elements of the output in that dimension will be equal to the
     # number of elements of any of them.
+    # TODO: We can decide to either specialize C code even further given the input types
+    #  Or make it general, regardless of whether static broadcastable information is given
+    init_dims = ""
     for i, candidates in enumerate(zip(*loop_orders)):
-        for j, candidate in enumerate(candidates):
-            if candidate != "x":
-                var = sub[f"lv{int(j)}"]
-                init_dims += f"dims[{i}] = {var}_n{candidate};\n"
-                break
-        else:
+        # TODO: Are candidates always either "x" or "i"? If that's the case we can
+        #  simplify some logic here. We don't need to track the `idx`
```

Comment:
```suggestion
        # TODO: Are candidates always either "x" or "i"? If that's the case, we can
        # simplify some logic here (e.g. we don't need to track the `idx`).
```

Reply:
The space in the second line is something that PyCharm does automatically for multi-line `TODO`s. Couldn't find if this was standard or specific to them